recipe-server/client/selfrepair/tests/test_normandy_driver.js

```diff
@@ -103,6 +104,35 @@ describe('Normandy Driver', () => {
       expect(client.isDefaultBrowser).toEqual(true);
       expect(client.searchEngine).toEqual('Yahoo');
     });
+
+    it('should pull plugin data from navigator.plugins', async () => {
+      const flashPlugin = {
+        name: 'Shockwave Flash',
+        description: 'Literally the worst',
```

Comment:
That's fair. This is actually going away now anyways, since #436 merged much sooner than I thought.